lib/configure.js

```diff
@@ -479,19 +479,32 @@ PythonFinder.prototype = {
   },
 
   failNoPython: function failNoPython () {
-    var errmsg =
-        'Can\'t find Python executable "' + this.python +
-        '", you can set the PYTHON env variable.'
-    this.callback(new Error(errmsg))
+    const err = new Error(
+      '\n******************************************************************\n' +
+      `node-gyp can't use "${this.python}",\n` +
+      'It is recommended that you install python 2.7, set the PYTHON env,\n' +
+      'or use the --python switch to point to a Python >= v2.6.0 & < 3.0.0.\n' +
+      'For more information consult the documentation at:\n' +
+      'https://github.com/nodejs/node-gyp#installation\n' +
+      '***********************************************************************'
+    );
+    err.noPython = true;
+    this.callback(err)
   },
 
   failPythonVersion: function failPythonVersion (badVersion) {
-    var errmsg =
-        'Python executable "' + this.python +
-        '" is v' + badVersion + ', which is not supported by gyp.\n' +
-        'You can pass the --python switch to point to ' +
-        'Python >= v2.5.0 & < 3.0.0.'
-    this.callback(new Error(errmsg))
+    const err = new Error(
+      '\n******************************************************************\n' +
+      `Python executable "${this.python}" is v${badVersion}\n` +
+      'this version is not supported by GYP and hence by node-gyp.\n' +
+      'It is recommended that you install python 2.7, set the PYTHON env,\n' +
+      'or use the --python switch to point to a Python >= v2.6.0 & < 3.0.0.\n' +
```

Comment:
Should this be 2.7.0 instead of 2.6.0 at this point?  [Python 2.6 has been EOL for 5 years](https://docs.python.org/devguide/index.html#branchstatus).